app/workers/bgs/submit_form686c_job.rb

```diff
@@ -11,29 +11,28 @@ class Invalid686cClaim < StandardError; end
 
     sidekiq_options retry: false
 
-    def perform(user_uuid, saved_claim_id, vet_info)
-      user = User.find(user_uuid)
-      Rails.logger.info('BGS::SubmitForm686cJob running!', { user_uuid:, saved_claim_id:, icn: user&.icn })
+    def perform(user_uuid, icn, saved_claim_id, vet_info)
+      Rails.logger.info('BGS::SubmitForm686cJob running!', { saved_claim_id:, icn: })
       in_progress_form = InProgressForm.find_by(form_id: FORM_ID, user_uuid:)
       in_progress_copy = in_progress_form_copy(in_progress_form)
       claim_data = valid_claim_data(saved_claim_id, vet_info)
       normalize_names_and_addresses!(claim_data)
-
-      BGS::Form686c.new(user).submit(claim_data)
+      temp_user = generate_temp_user(vet_info['veteran_information'])
```

Comment:
Same note applies as above re: calling this a `temp_user`

Reply:
changed to user_struct